### PR TITLE
Adding Madina's changes

### PR DESCRIPTION
**Pull request by MadinaHasan19**
Hi, I've added a new species of birds and thought it would be helpful to add it. Let me know if that's fine.
Thanks @MadinaHasan19

### CODE REVIEW

File: params/params_Madina.R [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 2
+    
+# e.g. "anas_krystallinus"    
+species.name <- "Laimoon "
```
Comment (ns-rse) [inline params/params_Madina.R]: I think there might be an extra space here.

```suggestion
species.name <- "Laimoon"
```

**Review by ns-rse — CHANGES_REQUESTED** [review]
Thanks @MadinaHasan19 I spotted an extra space and have made a suggestion to remove it.

**Review by ns-rse — APPROVED** [review]
That looks good to go, thanks @MadinaHasan19

**Comment by MadinaHasan19** [comment]
Thanks!

**Pull request merged**